engine-tests/src/test/java/org/terasology/rendering/nui/layers/mainMenu/savedGames/GameProviderTest.java

```diff
@@ -162,6 +178,15 @@ public void getNextGameNameDefaultNoSavesTest() {
         assertEquals(GAME_1, name);
     }
 
+    @Test
+    public void getNextGameNameCustomNoSavesTest() {
+        String gameName = "Custom";
+        final String name = GameProvider.getNextGameName(gameName);
+
+        assertNotNull(name);
```

Comment:
isn't this check included in the next one?

Reply:
No.we already have "saved" game  in next test . there - not.